index.html

```diff
@@ -172,7 +172,7 @@ <h2 class="wow fadeInLeftBig">Our Services</h2>
                   <!-- BEGIN SINGLE SERVICE -->
                   <div class="single_service wow fadeInLeft">
                     <div class="service_iconarea">
-                      <span class="fa fa-paper-plane service_icon"></span>
+                      <img src="img/services/InOut.png" height="150px">

```

Comment:
@PawanKolhe, you shouldn't use the inline styling ! You should make the icons dimensions to 128px or 150px !